ferrocene/doc/safety-manual/src/rustfmt/constraints.rst

```diff
@@ -0,0 +1,46 @@
+.. SPDX-License-Identifier: MIT OR Apache-2.0
+   SPDX-FileCopyrightText: The Ferrocene Developers
+
+.. default-domain:: qualification
+
+Constraints
+===========
+
+Install version control system
+------------------------------
+
+.. id:: RUSTFMT_CSTR_0010_HAVE_VCS
+
+Associated requirement ID: :id:`RUSTFMT_AVD_CHECK_DIFF_001`
+
+Before executing ``rustfmt``, the user shall ensure that the a version control
+system (VCS, e.g. ``git``) is installed and used.
```

Comment:
If we want to require a VCS (see next comment), having it installed shouldn't be a constraint, it should be a pre-requisite in the installation process.